pywbem_mock/_baseprovider.py

```diff
@@ -333,13 +295,114 @@ def remove_namespace(self, namespace):
                 CIM_ERR_NAMESPACE_NOT_EMPTY,
                 _format("Namespace {0!A} contains objects.", namespace))
 
+    @staticmethod
+    def interop_namespace_names():  # pylint: disable=no-self-use
+        """
+        Returns an iterable of the valid interop namespace names where
+        the names are considered case insensitive.
+
+        Returns:
+
+          List of :term:`string` containing the valid names for
+          namespaces that can be interop namespaces. Any name not on this list
+          cannot be an interop provider
+
+        """
+        return WBEMServer.INTEROP_NAMESPACES
+
+    def is_interop_namespace(self, namespace):  # pylint: disable=no-self-use
+        """
+        Tests if the namespace parameter defines a namespace name that is one
+        of the allowed set of DMTF names for the interop namespace.
+
+        This test is independent of the test in WBEMServer since it is just
+        confirming that there is an namespace with one of the valid interop
+        namespace names in the physical cim repository.
+
+        Parameters:
+
+            namespace (:term:`string`):
+                The namespace name that is to be tested.
+
+        Returns:
+            True: If the name is one of the valid interop namespace names
+            False: if the name is not an interop namespace valid name.
```

Comment:
Sphinx generates "Return type: True" from this. Add the return type bool, like so:

```
        Returns:
            bool: Indicates that the name is one of the valid interop namespace names.
```

Reply:
DONE